vignettes/install.Rmd

```diff
@@ -125,7 +125,17 @@ At this time, the following environment variables can be used to change the Rust
 The feature flags can be set by the `LIBR_POLARS_FEATURES` environment variable.
 For example, to enable the `full_features` feature, set the environment variable as like `LIBR_POLARS_FEATURES="full_features"`.
 
-Note that these features requires the Rust nightly toolchain `r rust_toolchain_version`.
+Currently `full_features` would work as a combination of the following features.
+
+- Features for CRAN compatibility
+  - `disable_auto_limit_max_threads`, this feature disables the automatic limit of the maximum number of threads to 2 for CRAN compatibility,
+    and the maximum number of threads is used by default.
+    In other words, if this feature is enabled, it is not compatible with CRAN.
```

Comment:
I think this warning is only for people that want to use `polars` in other packages and publish them to CRAN. This sentence could be less "scary" for standard R users